tests/accounts/test_csrf.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.test import Client
+from django.urls import reverse
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def _add_installed_apps(settings):
+    settings.LOGIN_URLS = [reverse("admin:login")]
+    settings.CSRF_FAILURE_VIEW = "maykin_common.accounts.csrf.csrf_failure"
+
+
+@pytest.mark.django_db
+def test_redirect_on_csrf_failure_if_logged_in(django_user_model):
+    user = django_user_model.objects.create_superuser(username="one")
+    client = Client(enforce_csrf_checks=True)
+    client.force_login(user)
+    response = client.post(reverse("admin:login"), {})
```

Comment:
```suggestion

    response = client.post(reverse("admin:login"), {})
```
please always use visual blocks for the

```
setup
setup

act

assert
assert
```